helm/ffc-pr-calculator/values.yaml

```diff
@@ -32,7 +27,6 @@ deployment:
   memoryThreshold: 80
 
 container:
-  imagePullPolicy: IfNotPresent
```

Comment:
These values are set by default in later versions of the helm library so no need to duplicate here